assopy/models.py

```diff
@@ -582,8 +582,8 @@ def __str__(self):
 
 
 class VatFare(models.Model):
-    fare = models.ForeignKey('conference.fare')
-    vat = models.ForeignKey(Vat)
+    fare = models.ForeignKey('conference.fare', on_delete=models.CASCADE)
+    vat = models.ForeignKey(Vat, on_delete=models.CASCADE)
```

Comment:
BTW, in general I'm more of a fan of SET_NULL or DO_NOTHING than CASCADE, but because cascade is default I guess it would require reviewing every single FK – so it's probably better to move them to explicit  CASCADE (for django20 warnings), like in this PR, and then maybe update them to some other values where it makes sense.